scheduling-manager/src/main/java/com/amazonaws/blox/scheduling/scheduler/engine/SingleTaskScheduler.java

```diff
@@ -19,22 +19,47 @@
 import com.amazonaws.blox.scheduling.state.ClusterSnapshot.ContainerInstance;
 import java.util.Arrays;
 import java.util.Collections;
+import java.util.Comparator;
 import java.util.List;
-import java.util.Map.Entry;
 
 /**
  * Temporary scheduler implementation that just starts a single task on the first instance in the
  * snapshot.
+ *
+ * <p>TODO Remove this, it's only a demo scheduler
  */
 public class SingleTaskScheduler implements Scheduler {
 
   @Override
   public List<SchedulingAction> schedule(ClusterSnapshot snapshot, EnvironmentVersion environment) {
```

Comment:
you'll have to create a new internal object for environmentversion or something similar because the returned type is now describeEnvironment which contains separate fields. can be done in the refactoring cr

Reply:
That's what I did for #278